.github/maintainers_guide.md

```diff
@@ -1,199 +1,162 @@
-# Maintainers Guide
+# Maintainer's Guide
 
-> ⚠️ This maintainers guide is out of date. Progress on updating it is being [tracked](https://github.com/slackapi/node-slack-sdk/issues/748).
+This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain this project. If you use this package within your own software but don't plan on modifying it, this guide is **not** for you. Please refer to the [Contributor's Guide](https://github.com/slackapi/node-slack-sdk/blob/main/.github/contributing.md).
 
-This document describes tools, tasks and workflow that one needs to be familiar with in order to effectively maintain
-this project. If you use this package within your own software but don't plan on modifying it, this guide is
-**not** for you.
+## 🛠 Tools
+Maintaining this project requires installing [Node.js](https://nodejs.org). All of the remaining tools are downloaded as `devDependencies`, which means you'll have them available once you run `npm install` in a working copy of this repository.
 
-## Tools
+## ✅ Tasks
 
-Maintaining this project requires installing [Node.js](https://nodejs.org) in your development environment. All of the
-remaining tools are downloaded as `devDependencies`, which means you'll have them available once you run `npm install`
-in a working copy of this repository.
+### ⚗️ Testing and Linting
+To run tests for a given package, navigate to the package directory and run `npm test`. To run linting, run `npm run lint`.
```

Comment:
🤦🏼‍♀️ You're totally right. I'll clean up the language to be clearer.

Reply:
I just now realized I misread your comment! I think you had it right: when running tests, you need to navigate to the individual package (ie, `web-api`, `events-api`, etc.).

Either way, this can do with more clarity, I'm sure, so will still tweak it a bit.